packages/terra-switch/NOTICE

```diff
@@ -0,0 +1,13 @@
+Copyright 2019 - 2020 Cerner Innovation, Inc.
```

Comment:
this should just be "2020" and not "2019 - 2020"

Reply:
updated : 48fce2d9ca73773ac6f9891feaa7f2a56b36f864